src/source/Crypto/IOBuffer.c

```diff
@@ -0,0 +1,101 @@
+#define LOG_CLASS "IOBuffer"
+#include "../Include_i.h"
+
+STATUS createIOBuffer(UINT32 initialCap, PIOBuffer* ppBuffer) {
+    STATUS retStatus = STATUS_SUCCESS;
+    PIOBuffer pBuffer = NULL;
+
+    pBuffer = MEMCALLOC(SIZEOF(IOBuffer), 1);
```

Comment:
Need to cast the result of MEMCALLOC to (PIoBuffer) type as compilers will warn

Reply:
Done.